lib/Form/Util/CustomFonts.php

```diff
@@ -87,12 +87,26 @@ public function enqueueStyle() {
       // When we load all custom fonts in one request, a form from WC Payments isn't displayed correctly.
       // It looks that the larger file size overloads the Stripe SDK.
       foreach (array_chunk(self::FONTS, self::FONT_CHUNK_SIZE) as $key => $fonts) {
-        $this->wp->wpEnqueueStyle('mailpoet_custom_fonts_' . $key, $this->generateLink($fonts));
+        $this->wp->wpEnqueueStyle('mailpoet_custom_fonts_' . $key, self::generateLink($fonts));
       }
     }
   }
 
-  private function generateLink(array $fonts): string {
+  public static function getCustomFontLink() {
+    $output = '';
+
+    foreach (array_chunk(self::FONTS, self::FONT_CHUNK_SIZE) as $key => $fonts) {
+      $output .= self::generateLink($fonts);
```

Comment:
You are concatenating URLs here without any delimiter. Each one needs to be wrapped in a separate `<link>` tag, otherwise it won't work.

Reply:
@wxa 
Not really.
The URL is not been concatenated here.

This will generate the full HTML link required in the `<link href>` element e.g the output would be `'https://fonts.googleapis.com/css?family=' . TOTAL_FONTS`

You can locate that here:  https://github.com/mailpoet/mailpoet/blob/1147ba6c8e6474ad9ed12c653d36365610fc56de/lib/Form/Util/CustomFonts.php#L109

This is the part that actually generates the HTML link: https://github.com/mailpoet/mailpoet/blob/1147ba6c8e6474ad9ed12c653d36365610fc56de/lib/Form/Util/CustomFonts.php#L105

Reply:
@wxa Yes, it is expected.

This makes use of one HTTP request to download all the required fonts. 

They are not multiple links but instead multiple font families, separated with a pipe character (|).

You can read more on that here: https://developers.google.com/fonts/docs/getting_started#overview

Reply:
@triple0t 

> Yes, it is expected.

> This makes use of one HTTP request to download all the required fonts.

> They are not multiple links but instead multiple font families, separated with a pipe character (|).

There is no pipe character between URLs. There are three links merged together without any delimiter. Let me mark the issue visually for you:

![image](https://user-images.githubusercontent.com/9365088/142178237-1ccba97f-526d-4ba2-868c-bbed4d5242d9.png)

Reply:
@wxa Wow, nice catch. Thanks for noticing this 🙌🏿.